### PR TITLE
Major refactoring

### PR DESCRIPTION
**Pull request by zelgerj**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,25 +2,62 @@ FROM alpine:edge
...
+    mv /usr/share/elasticsearch-2.4.2 /usr/share/elasticsearch && \
+    # prepare needed directories
+    mkdir -p /usr/share/elasticsearch/data/elasticsearch/nodes /usr/share/elasticsearch/logs /usr/share/elasticsearch/config/scripts && \
+    # install useful plugins
+    /usr/share/elasticsearch/bin/plugin install analysis-phonetic && \
+    /usr/share/elasticsearch/bin/plugin install analysis-icu && \
+
+    # install rabbit mq
+    curl -sSL https://www.rabbitmq.com/releases/rabbitmq-server/v3.6.1/rabbitmq-server-generic-unix-3.6.1.tar.xz | tar -xJ -C / --strip-components 1 && \
+    rm -rf /share/**/rabbitmq*.xz && \
```
Comment (sydekumf) [inline Dockerfile:38]: Consider enabling the rabbitmq management plugin: `rabbitmq-plugins enable rabbitmq_management`

File: tmp/usr/local/bin/setup-xdebug [hunk 2]
```diff
@@ -1,134 +0,0 @@
-#!/usr/bin/env bash
```
Comment (sydekumf) [inline tmp/usr/local/bin/setup-xdebug]: Please don't delete this file, we need it!

**Comment by zelgerj** [comment]
@sydekumf: thanks for the feedback. I've readded the xdebug bin and its documentation. Lets keep this change as a "refactoring" related topic. I would love to see a separate pull-request enhancing rabbitmq integration from your side 👍
@JSchlarb: Any feedback from your side?

**Comment by zelgerj** [comment]
@JSchlarb: ok i hope you're not drunk and are with us with this change 😋

**Pull request merged**